Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,903 @@
+                )
+                page += 1
+        elif demisto.command() == "dspm-get-asset-details":
+            return_results(get_asset_details(client, demisto.args()))
+        elif demisto.command() == "dspm-get-asset-files-by-id":
+            return_results(get_asset_files_by_id(client, demisto.args()))
+        elif demisto.command() == "dspm-get-data-types":
+            return_results(get_data_types(client))
+        elif demisto.command() == "dspm-get-data-types-findings":
+            page = 0
+            all_data_type_findings = []
+
+            while True:
+                data_type_findings = get_data_type_findings(
+                    client, demisto.args(), page
+                )
+                if not data_type_findings:
+                    if page == 0:
+                        readable_output = (
+                            "### Data Types\n"
+                            "| No | Key |\n"
+                            "|----|-----|\n"
+                            "**No entries.**\n"
+                        )
+                        return_results(
+                            CommandResults(
+                                outputs_prefix="DSPM.DataTypesFindings",
+                                outputs_key_field="Key",
+                                outputs=[],
+                                readable_output=readable_output,
+                            )
+                        )
+                    break
+
+                all_data_type_findings.extend(data_type_findings)
+
+                data_type_findings_formatted = [
+                    {
+                        "No": index + 1 + (page * MAX_PAGE_SIZE),
+                        "Key": dt["dataTypeName"],
+                    }
+                    for index, dt in enumerate(data_type_findings)
+                ]
+
+                readable_output = "### Data Types\n | No | Key  |\n |----|------|\n"
+                for item in data_type_findings_formatted:
+                    readable_output += f"| {item['No']}  | {item['Key']} |\n"
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.DataTypesFindings",
+                        outputs_key_field="Key",
+                        outputs=data_type_findings_formatted,
+                        readable_output=readable_output,
+                    )
+                )
+
+                page += 1
+        elif demisto.command() == "dspm-update-risk-finding-status":
+            return_results(update_risk_finding_status(client, demisto.args()))
+        elif demisto.command() == "dspm-get-list-of-alerts":
+            page = 0
+            while True:
+                alerts = get_list_of_alerts(client, demisto.args(), page)
+                if not alerts:
+                    if page == 0:
+                        readable_output = (
+                            "### Data Types\n"
+                            "| No | Key |\n"
```

Comment:
please use tableToMarkdownm see this article:
https://xsoar.pan.dev/docs/integrations/code-conventions#tabletomarkdown

Reply:
Sure